recipe/run_test.py

```diff
@@ -8,7 +8,7 @@
 # See https://github.com/conda-forge/gdal-feedstock/issues/131
 from osgeo.gdal_array import *
 
-drivers = ['netCDF', 'HDF4', 'HDF5', 'GTiff', 'PNG', 'JPEG', 'GPKG', 'KEA', 'JP2OpenJPEG', 'WCS']
+drivers = ['netCDF', 'HDF4', 'HDF5', 'GTiff', 'PNG', 'JPEG', 'GPKG', 'KEA', 'JPEG', 'WCS']
```

Comment:
@gillins is this OK?

Reply:
We already have `JPEG` in this list. Is there a problem with the `JP2OpenJPEG` driver?

Reply:
@ocefpaf I'm a bit confused about why you are dropping the test for `JP2OpenJPEG` (the JPEG2000 driver based on `openjpeg`)?